database.sql

```diff
@@ -0,0 +1,6 @@
+create table github_discord_url(
```

Comment:
this file was already reviewed - it should not be appearing here for review again